src/main/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysis.java

```diff
@@ -312,16 +361,20 @@ private PostContingencyResult runPostContingencySimulation(LfNetwork network, Ac
         });
 
         LfContingency.reactivateEquations(deactivatedEquations, deactivatedEquationTerms);
-
         stopwatch.stop();
         LOGGER.info("Post contingency '{}' simulation done in {} ms", lfContingency.getContingency().getId(),
                 stopwatch.elapsed(TimeUnit.MILLISECONDS));
 
-        return new PostContingencyResult(lfContingency.getContingency(), postContingencyComputationOk, new ArrayList<>(postContingencyLimitViolations.values()));
+        return new PostContingencyResult(lfContingency.getContingency(), postContingencyComputationOk, new ArrayList<>(postContingencyLimitViolations.values()),
+            branchResults.stream().collect(Collectors.toMap(BranchResult::getBranchId, Function.identity())),
```

Comment:
Same here, maybe it would be simpler to change the constructor so that we just provide the lists